services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.test.tsx

```diff
@@ -78,7 +78,47 @@ describe('Release Dialog', () => {
             ],
             expect_message: true,
             expect_queues: 0,
-            data_length: 1,
+            data_length: 2,
+            teamName: '',
+        },
+        {
+            name: 'normal release with deploymentMetadata set',
+            props: {
+                app: 'test1',
+                version: 2,
```

Comment:
for the commit message (when merging), I would suggest:

> Display person who triggered deployment
> 
> Display the author and time of deployment in the release dialog.

Reply:
Will change it when I merge, thanks !